src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/EventServiceImplementation.java

```diff
@@ -0,0 +1,39 @@
+package ro.societateahermes.backendservice.service.serviceImplementation;
+
+import net.sf.saxon.functions.ConstantFunction;
+import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.Event;
+import ro.societateahermes.backendservice.repository.EventRepositoryInterface;
+import ro.societateahermes.backendservice.service.dto.NotificationSwitchDTO;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Service
+public class EventServiceImplementation {
+
+    private final EventRepositoryInterface eventRepository;
+
+    public EventServiceImplementation(EventRepositoryInterface eventRepository) {
+        this.eventRepository = eventRepository;
+    }
+
+    public NotificationSwitchDTO getEventStatusByEventName(String eventName) {
+        Optional<Event> eventOptional = eventRepository.findByEventName(eventName);
+        if (eventOptional.isPresent()) {
+            Event event = eventOptional.get();
+            if (event.getEventStartDate().isAfter(LocalDateTime.now())) {
+                // has not started yet
```

Comment:
a method should never contain comments, only when the implementation is not clear on purpose due to a workaround for a bug, the code should always explain itself